### DISCUSSION TITLE
Welcome to GBA-PK-multiplayer Discussions!

### DISCUSSION THREAD

**Discussion by TheHunterManX** [TheHunterManX/GBA-PK-multiplayer#70] [Announcements]
Hello. This is a place I have opened because I have noticed people wanting it. You can discuss issues you are having that aren't bugs/glitches.

**Comment by Tibbysan** [comment]
![Screenshot 2024-08-31 150955](https://github.com/user-attachments/assets/9b0f150a-af58-4a1f-90bf-24e7bbc15ac1)
![Screenshot 2024-08-31 150949](https://github.com/user-attachments/assets/e01da711-a165-468f-840c-e645960b5761)

Some reason when I link both Ruby (EN) and Emerald (EN) together they give me errors.

**Comment by Tibbysan** [comment]
I also just relized the 3.5 version you put up out seems different then the one in the new GBA-PK Installation Guide Update video.

![image](https://github.com/user-attachments/assets/9a35035f-dfba-48d7-ba85-a85102546566)

**Reply by TheHunterManX** [reply]
The reason why is because I did not update the main repository before, and only put the files in release. It should now be all updated. You are using Alpha 3.0 in your photos, as there are only ~11000 lines, yet the errors are in lines > 20000, as well as players being classes now instead of variables.

**Reply by Tibbysan** [reply]
I am using the 3.5 folder you placed here, not version 3. :) I deleted my version 3.

**Reply by TheHunterManX** [reply]
As stated, the main repository was not updated at the time you downloaded it. Please try to redownload the files

**Reply by Tibbysan** [reply]
My apologie, I miss read. :)

**Reply by Tibbysan** [reply]
Last thing, not sure if this is because it's Ruby connecting to Emerald but Player May has this werid hair dye thing going on in Emerald  version, but Rival May seems fine.

![image](https://github.com/user-attachments/assets/0ff6073e-25d7-445c-9621-9ec2f14b5fb7)

**Comment by TheHunterManX** [comment]
This is due to how palettes are handled between genders. Male and Female palettes are slightly different, hence the hair color. The reason why NPC may looks ok is because she is using a different palette from your character. But fixing it requires messing with palettes, which I will do eventually

**Reply by Tibbysan** [reply]
Are you still planing on working on this? :)

**Comment by BenPhoenix-24** [comment]
A Question regarding Legendary Pokemons and Gyms:

Do Each Player get to Battle the Gym and Legendary, Unique Pokemon Separately or They have to share the Legendary Pokemon per Server?
If 3 Players are Playing on a Server, Will there exist Three Articuno, Moltres and Zapdos, Six Snorlax in that Server or 1, 1, 1 and 2 in that Server respectively?

**Comment by neolakshay** [comment]
**Need HELP**

I got it to work on my pc , BUT I can't send input to both game windows at same time , only one windows stay active and the other one stops receiving inputs from keyboard ,, how do I fix it?????
